Update FW for nPM1304 (#268)
NordicSemiconductor/pc-nrfconnect-npm: Changelog.md

## 2.2.1 - Unreleased

### Changed

> -   Default battery model in FW for nPM1304 to LP18917.

Reviewer: ```suggestion
-   Default battery model in the firmware for nPM1304 to LP18917.
```

### Fixed

> -   Fixed issue in FW for nPM1304 where battery measurements could stop while
>     charging.

Reviewer: ```suggestion
-   Issue in the firmware for nPM1304 where battery measurements would stop while
    the battery was charging.
```
